client/src/components/create-collection-form/form-steps/LayerImageUpload.tsx

```diff
@@ -137,19 +143,34 @@ const LayerImageUpload = ({
           );
         })}
       </Box>
-      <Paper>
-        <Input
-          type="number"
-          value={state.generative.quantity}
-          onChange={handleQuantityChange}
-          placeholder="0"
-          label="Collection Quantity"
-          InputProps={{
-            inputProps: { min: 1, max: getMaxQuantity() },
-          }}
-          required
-        />
-      </Paper>
+      <Collapse in={getMaxQuantity() > 0}>
+        <p>
+          Maximum possible collection quantity: {getMaxQuantity()}.{" "}
```

Comment:
Close p tag on this line otherwise error

Reply:
Also please replace <p> tags with <Typography> from MUI for consistency